src/Time/Tags.hpp

```diff
@@ -46,10 +46,20 @@ struct TimeStep : db::SimpleTag {
 
 /// \ingroup DataBoxTagsGroup
 /// \ingroup TimeGroup
-/// \brief Tag for compute item for ::Time of the current substep (from
-/// TimeStepId)
-struct SubstepTime : db::ComputeTag {
-  static std::string name() noexcept { return "SubstepTime"; }
+/// \brief Tag for ::Time of the current substep
+///
+/// \see SubstepTimeCompute
+struct SubstepTime : db::SimpleTag {
+  using type = ::Time;
+};
+
+/// \ingroup DataBoxTagsGroup
+/// \ingroup TimeGroup
+/// \brief Tag for computing the substep time from (from `Tags::TimeStepId`)
+///
+/// \see SubstepTime
+struct SubstepTimeCompute : SubstepTime, db::ComputeTag {
+  static std::string name() noexcept { return db::tag_name<SubstepTime>(); }
```

Comment:
should the `name` function be in the `SimpleTag`?  Oh, wait, I see, I don't need to specify `name` in the `SimpleTag` because `db::tag_name` will do the right thing?